packages/web/app/views/programRegistry/ProgramRegistryTable.jsx

```diff
@@ -125,9 +125,7 @@ export const ProgramRegistryTable = ({ searchParameters }) => {
       {
         key: 'clinicalStatus',
         title: <TranslatedText stringId="programRegistry.clinicalStatus.label" fallback="Status" />,
-        accessor: row => {
```

Comment:
converting this from an accessor didn't make a different in terms of the bug but seemed more consistent with how CellComponents are used everywhere else.